ntbs-service/Pages/Notifications/Edit/ContactTracing.cshtml.cs

```diff
@@ -22,14 +22,17 @@ public ContactTracingModel(INotificationService service, IAuditService auditServ
         [BindProperty]
         public ContactTracing ContactTracing { get; set; }
 
-        public override async Task<IActionResult> OnGetAsync(int? id)
+        public override async Task<IActionResult> OnGetAsync(int? id, bool isBeingSubmitted)
         {
             Notification = await service.GetNotificationAsync(id);
             if (Notification == null)
             {
                 return NotFound();
             }
 
+            NotificationId = Notification.NotificationId;
+            NotificationStatus = Notification.NotificationStatus;
+            Notification.SetFullValidation(NotificationStatus, isBeingSubmitted);
             ContactTracing = Notification.ContactTracing;

```

Comment:
Do we not need to call `.SetFullValidation` on ContactTracing? I'm guessing not, since it doesn't have any fields it affects, but I wonder if we're loosing out on some commonisation possibilities because of that?

Reply:
FullValidation is then passed as flag onto next page through url parameter, if we omit this then the page that we will navigate from Contact Tracing page won't know that the form is being submitted and so the next page.

I thought about commonising this. Although commonisation won't help much in terms of saving duplication, but will help in making sure that Some flags are set

Reply:
Agree we should definitely be setting it on Notification - was just wandering if we wanted to set it on contact tracing too, for consistency!